app/components/Aside.css.ts

```diff
@@ -0,0 +1,19 @@
+import { style } from "@vanilla-extract/css";
+
+export const aside = style({
```

Comment:
(How do you feel about using the CSS format for CSS declarations (for example using CSS Modules)?)

Reply:
I don't mind it - and actually do that in [my personal site (CSS modules)](https://github.com/JoshuaKGoldberg/dot-com/blob/main/src/components/Squiggly.module.css) and the [Philly JS Club site (raw `.css`)](https://github.com/philly-js-club/philly-js-club-website/blob/ecb5508361589c8783448834274dbbff884a8063/app/shared.css). My big gripe with it is that it's not type-safe 😬. Vanilla Extract is the closest thing to type-safe CSS modules I've been able to find.

Long-term I'd love to have a fully built up type-safe styling system: https://github.com/philly-js-club/philly-js-club-website/issues/84. But those options are all still in-flux. 😞